packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -207,6 +207,8 @@ enum _PickerColumnType {
   minute,
   // AM/PM column in time and dateAndTime mode.
   dayPeriod,
+  // Two points column in time and dateAndTime mode.
```

Comment:
```suggestion
  // Time separator column in time and dateAndTime mode.
```
It is better to refer to the separator by a generic name. We do not have to do so in this PR, but we will want to later localize the separator so it shows the right string for locals that don't use ":".